### ISSUE TITLE
Cannot run out of the box google collab pixelNERF

### ISSUE THREAD

**Issue by 1ssb** [airalcorn2/pytorch-nerf#8]
There seems to a data.zip file for pixel Nerf implementation. Do you mind listing out the steps to get it and more importantly a list of steps to perform inference on a new image, directly from the collab?

**Comment by airalcorn2** [comment]
The instructions for generating the ShapeNet datasets can be found [in the README](https://github.com/airalcorn2/pytorch-nerf#generating-the-shapenet-datasets). The final cell of the Colab notebook demonstrates how to perform inference on a test image and viewpoint:

```python3
test_W_i = E(test_source_image)
(_, C_rs_f) = run_one_iter_of_pixelnerf(
    test_ds,
    N_c,
    t_i_c_bin_edges,
    t_i_c_gap,
    test_os,
    camera_distance,
    scale,
    test_W_i,
    chunk_size,
    F_c,
    N_f,
    t_f,
    N_d,
    d_std,
    t_n,
    F_f,
)
plt.imshow(C_rs_f.detach().cpu().numpy())
```

**Comment by 1ssb** [comment]
Hi Michael, the very first block of code is not working, the data.zip has
not been provided directly, is the shapenet data, the data.zip?

**Comment by airalcorn2** [comment]
You need to generate the data yourself using the instructions provided in the README, zip the generated `data` folder, and then put that in your own Google Drive.

**Comment by 1ssb** [comment]
Thanks, how would you like to be cited?

**Comment by airalcorn2** [comment]
No citation necessary. You can just link to this repository in any projects that build off of it.

**Comment by 1ssb** [comment]
Hi is there a way to train the pixelNerf using this implementation?

**Comment by airalcorn2** [comment]
The script/notebook trains a pixelNeRF.

**Comment by 1ssb** [comment]
Awesome, thanks!

**Comment by 1ssb** [comment]
Hi, I do not understand how to use the images themselves instead of the
images of shapenet with masks. Is there a way to do it directly without the
Metadata of the images?

**Comment by Reznox7** [comment]
@airalcorn2 Can we train custom datasets using any of the approaches you provided?
